lib/SVG/TT/Graph/HeatMap.pm

```diff
@@ -0,0 +1,929 @@
+package SVG::TT::Graph::HeatMap;
+
+use Modern::Perl;
+use Carp;
+use Data::Dumper;
+use SVG::TT::Graph;
+use base qw(SVG::TT::Graph);
+use vars qw($VERSION $TEMPLATE_FH);
+$VERSION     = $SVG::TT::Graph::VERSION;
+$TEMPLATE_FH = \*DATA;
+
+
+=head1 NAME
+
+SVG::TT::Graph::XY - Create presentation quality SVG line graphs of XY data points easily
+
+=head1 SYNOPSIS
+
+  use SVG::TT::Graph::XY;
+
+  my @data_cpu  = (0.3, 23, 0.5, 54, 1.0, 67, 1.8, 12);
+  my @data_disk = (0.45, 12, 0.51, 26, 0.53, 23);
+
+  my $graph = SVG::TT::Graph::XY->new({
+    'height' => '500',
+    'width'  => '300',
+  });
+
+  $graph->add_data({
+    'data'  => \@data_cpu,
+    'title' => 'CPU',
+  });
+
+  $graph->add_data({
+    'data'  => \@data_disk,
+    'title' => 'Disk',
+  });
+
+  print "Content-type: image/svg+xml\n\n";
+  print $graph->burn();
+
+=head1 DESCRIPTION
+
+This object aims to allow you to easily create high quality
+SVG line graphs of XY data. You can either use the default style sheet
```

Comment:
and it is neither a XY plot, I will have to work on the documentation before I would even feel good about letting this go up to CPAN. (sorry)